recipes/veloxchem/bld.bat

```diff
@@ -0,0 +1,40 @@
+setlocal EnableDelayedExpansion
+
+:: configure!
+cmake ^
+    -S"%SRC_DIR%" ^
+    -Bbuild ^
+    -GNinja ^
+    -DCMAKE_BUILD_TYPE:STRING=Release ^
+    -DCMAKE_INSTALL_PREFIX:PATH="%PREFIX%" ^
+    -DCMAKE_CXX_COMPILER:STRING=clang-cl ^
+    -DCMAKE_C_COMPILER:STRING=clang-cl ^
+    -DENABLE_ARCH_FLAGS:BOOL=OFF ^
+    -DVLX_LA_VENDOR:STRING="Generic" ^
+    -DPython_EXECUTABLE:STRING="%PYTHON%" ^
+    -DPYMOD_INSTALL_FULLDIR:PATH="Lib\site-packages\veloxchem"
+if errorlevel 1 exit 1
+
+:: build!
+cmake --build build --config Release --parallel %CPU_COUNT% -- -v -d stats
+if errorlevel 1 exit 1
+
+:: test!
+set KMP_DUPLICATE_LIB_OK=TRUE
+:: we only run unit tests here, integration tests are run later on
+cd %SRC_DIR%\build
+cmake --build . --config Release --parallel %CPU_COUNT% --target utests -- -v -d stats
+ctest -L unit --output-on-failure --parallel %CPU_COUNT%
```

Comment:
Can you reduce the number of tests? This build is quite long and it's more of the upstream developers responsibility to ensure that their package is correct. Here we only want to know if it was built/packaged correctly.

Reply:
I'm now skipping the unit tests entirely and only running a subset of the integration tests. Hopefully things speed up a bit.